packages/components/src/DataTable/BodyLoading.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Table } from "@tanstack/react-table";
+import { Glimmer } from "../Glimmer";
+
+interface BodyLoadingProps<T> {
+  table: Table<T>;
+}
+
+export function BodyLoading<T extends object>({ table }: BodyLoadingProps<T>) {
+  const loaderRows = table.getState().pagination.pageSize;
```

Comment:
@rodrigoeidelvein Made the changes asked for. One more thing that came up is whether we should consider putting a cap on the maximum amount of glimmers that get shown. For example if someone decided their minimum items per page was 1000, should we cap the amount of loaders that we show at 100? 150? 200? Or do you think this is not relevant?

Reply:
I don't think we need it. If someone is putting 1000 items per page, they're aware of the performance problems they'll encounter 😆